File: Lista2/C++/Kwadrat.cpp
#include <iostream>
#include <cmath>
#include "Kwadrat.h"

Kwadrat::Kwadrat(double bok)
{
    this->bok = bok;
}

double Kwadrat::obwod()
{
    return (4 * bok);
}

double Kwadrat::pole()
{
    return (pow(bok, 2));
}

Kwadrat::~Kwadrat()
{

}

File: Lista2/Java/Kwadrat.java
public class Kwadrat extends Czworokat
{
    private double bok;

    Kwadrat(double bok)
    {
        this.bok = bok;   
    }

    public double obwod()
    {
        return (4 * bok);
    }

    public double pole()
    {
        return (Math.pow(bok, 2));
    }
}